instruments/thorlabs/thorlabsapt.py

```diff
@@ -1286,6 +1343,7 @@ def status_bits(self):
             )
             # The documentation claims there are 14 data bytes, but it seems
             # there are sometimes some extra random ones...
+            # fixme: wrong expected datatype? MOT_GET_STATUSUPDATE expected
             resp_data = self._apt.querypacket(
                 pkt,
                 expect=_cmds.ThorLabsCommands.MOT_GET_POSCOUNTER,
```

Comment:
The expected package seems wrong, but I cannot test the routine with hardware. Therefore I left a note here... would be great to test and see if correct or not. @scasagrande: Ideas / recommendations?

Reply:
That's a good question..........and I have no idea 😄 

I also don't have access to the hardware so I can't help there. Maybe we just run with it and see what happens?

Reply:
Agreed. I put the note with the `fixme` in there for the future, i.e., if somebody complains that it doesn't work. Then we can use git blame, find this PR, and see all the notes - hopefully ;)

I think the controllers that are supported by this command are older models, or at least different models from what I'm often using, so I don't have access to the hardware either.  For that reason, the `.. note::` is in the docstring of this function, see above (with badly formatted commenting...).